apps/nowcasting-app/components/map/sites-zoom-slider.tsx

```diff
@@ -1,14 +1,10 @@
 import useGlobalState from "../helpers/globalState";
-import useState from "react";
 
-const SitesMapSlider: React.FC<{ selected?: string; unselected?: string }> = ({
+const SitesMapSlider: React.FC<{ selected?: string; unselected?: string; event?: any }> = ({
```

Comment:
Hey @braddf, This is what I came up with for the initial "slider" for the map. Would be great to have feedback on the ranges for the zoom levels (line 13-22) and if this is sort of what you were thinking. I know it isn't a traditional "slider".  I felt like keeping the look boxy fit with the design. When you click the `+/-` buttons, the highlighted view changes. 

As you can see, I tried adding an event handler to each div, but Typescript seemed unhappy with the `event.target` situation when I pushed my code. That's why it's been deleted. Didn't seem like it was necessary for this commit. 

Any other feedback, pointers are welcome.

Reply:
Great - just making sure I've been through all these comments, sorry for the delay... we can take a look at this design on the next iteration when we finish the aggregation stuff; I have some ideas to make this a little less intrusive, but we'll have to play around and see what feels right 👍